### ISSUE TITLE
Response headers

### ISSUE THREAD

**Issue by SAbadi** [devlucky/Kakapo.js#132]
Hello,

Setting Response's headers is not possible using the latest version of Kakapo(v0.2.1) although it is mentioned in the documentation like below: 

``` javascript
import { Server, Response, Router } from 'kakapo';

const server = new Server();
const router = new Router();

router.get('/users', request => {
  const code = 400;
  const body = {};
  const headers = {};

  return new Response(code, body, headers);
});

server.use(router);
```

I guess this is probably the same kind of issue as Kakapo(v0.2.0) previously had with Response's status code.

**Comment by zzarcon** [comment]
@SAbadi can you provide the code you are using to do the request?

**Comment by SAbadi** [comment]
@zzarcon 

Here is my code:

```javascript
return new Kakapo.Response(200, {}, {'X-request-date': new Date().getTime()});
```

Basically header object which is supposed to be the third parameter in the Response's constructor(based on the documentation) gets ignored totally.  By default the only header which is set for the Response is:

content-type: application/json; charset=utf-8

And its not possible to add more headers or change any of the headers. The issue is in xhrInterceptor.js which there is no usage of KakapoResponse.headers.

I can see "fakeHeaders" object(line 17) doesn't get updated by any means, so that's why Response's headers is always only the default value.

By the way I realized that Response's statusText is always empty string.

**Comment by zzarcon** [comment]
@SAbadi Fixed in `0.2.3`, please let me know if works fine for you now :)
